scripts/addHeaderToSource.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/**
+ * The purpose of this build script is to take a file on disk and to add a header to it.
+ * This header contains the license and a version number which corresponds to the latest 
+ * git tag and commit.
+ */
+
+const path = require('path');
+const fs = require('fs');
+const getStdin = require('get-stdin');
+
+
+getStdin()
+.then(source => {
+  const templateVersion = require('./util/templateVersion');
+  const VERSION_TEMPLATE_STRING = require('./util/versionTemplateString');
+
+  const pathLicense = path.join(process.cwd(), 'LICENSE.txt');
```

Comment:
@minasmart I wish we could do that 😢 This script executes each time on rebuild since it's not `babel` watch thats building.